.teamcity/_self/projects/WPComTests.kt

```diff
@@ -198,6 +198,15 @@ fun jetpackBuildType(screenSize: String): BuildType {
 		name = "Jetpack tests ($screenSize)"
 		description = "Runs Calypso Jetpack E2E tests using $screenSize screen resolution"
 
+		params {
+			select(
+				name = "JETPACKHOST",
+				value = "WPCOM",
+				label = "Jetpack Host",
+				options = listOf("WPCOM","PRESSABLE","PRESSABLEBLEEDINGEDGE")
```

Comment:
I'm not familiar with TeamCity config files, but if you plan to run only 1 type of Jetpack E2E, `JETPACKHOST` can be set statically to `PRESSABLEBLEEDINGEDGE` That's what current CircleCI is using.

Reply:
I can set the default to `PRESSABLEBLEEDINGEDGE`, or just remove the config option entirely, what do you think?

Reply:
I think the approach in e3ebab9 should work well.